nature/api.py

```diff
@@ -125,7 +124,7 @@ class Meta:
                   'criteria', 'conservation_programmes')
 
 
-class FeatureSerializer(ProtectedHyperlinkedModelSerializer, GeoModelSerializer):
+class FeatureSerializer(ProtectedHyperlinkedModelSerializer):
```

Comment:
We need need to specify it's a `GeoModelSerializer`. After it's added in `INSTALLED_APP`, it will automatically serialize the geometry field to GeoJSON.

Reply:
That's sweet, I think I've used this in some other projects but forgot it here :)